src/profile/components/Settings/Password/ErrorMessage.tsx

```diff
@@ -5,16 +5,14 @@ interface IProps {
   errors?: string[];
 }
 
-export const ErrorMessage: FC<IProps> = ({ errors }) => {
-  return (
-    <>
-      {errors
-        ? errors.map((error) => (
-            <li className={style.errorMessage}>
-              <ol>{error}</ol>
-            </li>
-          ))
-        : null}
-    </>
-  );
-};
+export const ErrorMessage: FC<IProps> = ({ errors }) => (
+  <>
+    {errors
+      ? errors.map((error) => (
+          <li className={style.errorMessage}>
+            <ol>{error}</ol>
+          </li>
+        ))
+      : null}
+  </>
+);
```

Comment:
There is really no real reason to do this?

If we were to change something I would rather just remove the fragments:
``` typescript
export const ErrorMessage: FC<IProps> = ({ errors }) => {
  return errors
    ? errors.map((error) => (
        <li className={style.errorMessage}>
          <ol>{error}</ol>
        </li>
      ))
    : null;
};
// Or some formatting like that I guess
```